### ISSUE TITLE
Custom labels and annotations to be added to Pod template of a Gatling runner job

### ISSUE THREAD

**Issue by yokawasa** [st-tech/gatling-operator#62]
# Feature request

It would be great if we could add custom  labels and annotations to Pods create from Gatling Runner Job.

## How to achieve it?

I can come up with the following 2 ways: 

#### 1. Extends all `.metadata.labels` and `.metadata.annotations` defined in Gatling CR to Gatling runner pods

For example, if you define the labels and annotations in Gatling CR like this, the same labels and annotations are added to relevant Gatling runner pods 

```yaml
apiVersion: gatling-operator.tech.zozo.com/v1alpha1
kind: Gatling
metadata:
  name: gatling-sample01
  ############# this part ###############
  labels:
    app: my-sample-app
  annotations:
    gatling-operator.tech.zozo.com/hello: "true"
  ############# end this part ###############
```

#### 2. Add a new `ObjectMeta` field to Gatling CR and allow to add to define labels and annotations in it like this:

```yaml
apiVersion: gatling-operator.tech.zozo.com/v1alpha1
kind: Gatling
metadata:
  name: gatling-sample01
spec:
  ############# this part ###############
  objectMeta: 
    labels:
       app: my-sample-app
    annotations:
      gatling-operator.tech.zozo.com/hello: "true"
  ############# end this part ###############
  podSpec:
    serviceAccountName: "gatling-operator-worker"
    gatlingImage: ghcr.io/st-tech/gatling:latest          # Optional. Default: ghcr.io/st-tech/gatling:latest. The image that will be used for Gatling container.
    rcloneImage: rclone/rclone   
```

These are just ideas that I can think of now. There may be a better way to achieve it.

**Comment by yokawasa** [comment]
assigned to @S-mishina

**Comment by yokawasa** [comment]
Closing the issue as the relevant PR has been merged #63 
@S-mishina Thanks for the contribution!
